.travis.yml

```diff
@@ -73,6 +79,21 @@ install:
   # make sure the sqlite file exists to avoid race conditions
   - python -c "from bluesky.utils import get_history; get_history()"
 
+before_script:
+  - |
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+        brew cask install homebrew/cask-versions/adoptopenjdk8
+        brew install kafka
+        brew services start zookeeper
+        brew services start kafka
+        sleep 60
```

Comment:
Can we also check the statuses of the services and see if they started instead of that long wait? The same may apply to the Linux's Docker approach.

Reply:
I don't know how to do that

Reply:
`brew services status` and `grep`/parse it?